views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -294,9 +311,30 @@ define([
                 this._interruptRecording();
 
                 provider.cancel();
+                setState(recorder, recorderStates.CANCEL);
```

Comment:
Caution, the `setState()` function is also triggering an event using the state as event name.
So this will result in a duplication of `cancel` event. (`recorder.trigger('cancel')` will be called twice)

Reply:
fixed, changed the name